README.md

```diff
@@ -77,7 +77,7 @@ manim -p -ql example.py SquareToCircle
 ```
 
 You should see your native video player program pop up and play a simple scene in which a square is transformed into a circle. You may find some more simple examples within this
-[GitHub repository](master/example_scenes). You can also visit the [official gallery](https://docs.manim.community/en/stable/examples.html) for more advanced examples.
+[GitHub repository](https://github.com/ManimCommunity/manim/tree/master/example_scenes). You can also visit the [official gallery](https://docs.manim.community/en/stable/examples.html) for more advanced examples.
```

Comment:
I think that making it link to either `/example_scenes` or `example_scenes` (not sure which) would actually work. So essentially just removing `master`. You'll have to try it tho

Reply:
@WampyCakes thanks for the hint. You are right, changing it to just **example_scenes** works fine